graylog2-web-interface/src/components/cluster/index.js

```diff
@@ -0,0 +1 @@
+export GraylogClusterOverview from './GraylogClusterOverview';
```

Comment:
This way of reexporting the default export of an ES6 module is not working in some ES6 implementations. Therefore we changed this at some point to follow [a form which works in all implementations](https://github.com/Graylog2/graylog2-server/blob/master/graylog2-web-interface/src/components/common/index.jsx).